FEATURES.md

```diff
@@ -6,3 +6,20 @@ To close Helm-dashboard, click on the button in the rightmost corner of the scre
 # Multicluster
 If you want to switch to a different cluster, simply click on the corresponding cluster as shown in the figure. [Click here](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) to learn how to access multiple clusters.
 ![Multicluster_screenshot](images/screenshot_multicluster.png)
+
+# Repository
+Essentially, a repository is a location where charts are gathered and can be shared. If you want to learn more about repositories, [click here](https://helm.sh/docs/topics/chart_repository/). You can find the repository in the home section, as depicted in the figure.
+![Repository3](images/screenshot_repository3.png)
+
+You can add the repository by clicking on 'Add Repository', as shown in the figure.
+![Repository](images/screenshot_repository.png)
+
+After completing that step, enter the following data: the repository name and its URL. You can also add the username and password, although this is optional.
+![Repository2](images/screenshot_repository2.png)
+
+If you want to update your repository, delete it, and then filter your chart, you can do so as shown in the figure.
```

Comment:
This part is a bit meaningless. Updating repository is a separate action, not connected to deletion and filtering. Deletion is again a separate action and deserves own couple of lines of doc. Filtering is a whole own story, related to how you browse the list of available charts, that you can filter that list and trigger installing of chart. 

Bottom line: describe individual features of repository actions, not whole block/

Reply:
so I need to write about filter, update and delete separately only ? all other things are okay?

Reply:
Yes, you have improved the above pieces and those are fine. Good job!

We just need to improve the remaining parts.